webpack.config.js

```diff
@@ -90,8 +90,7 @@ module.exports = {
     historyApiFallback: {
       index: 'dev.html',
     },
-    disableHostCheck: true,
-    compress: true,
+    allowedHosts: 'all',
```

Comment:
Which host does this refer to?

Reply:
`allowHosts: 'all'` is basically the same as `disableHostChekc: true`. This update was made as part of migrating `webpack-dev-server` to v4. You can find about it in [this migration guide](https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md).